scripts/consolidate-hyperindex-docs.js

```
@@ -0,0 +1,420 @@
+    );
+
+    // Create a mock environment for the sidebar evaluation
+    const mockRequire = (modulePath) => {
+      if (modulePath === "./supported-networks.json") {
+        return supportedNetworks;
+      }
+      throw new Error(`Unexpected require: ${modulePath}`);
+    };
+
+    // Evaluate the sidebar configuration
+    const sidebarConfig = eval(`(function() {
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)